test/kuttl/testmetrics/metrics-provider-dummy/logs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
if you use my suggestion, you do not need this you can just assert for metrics and analysis status

Reply:
Oh Alright I will remove this then.